packages/i18nify-js/src/modules/phoneNumber/data/phoneFormatterMapper.ts

```diff
@@ -1,251 +1,231 @@
-  TH: 'xxx-xxxxxxx',
-  TJ: 'xxx xx xx xx',
-  TL: 'xxx-xxxxxxx',
-  TN: 'xx xxxxxx',
-  TR: 'xxx xxx xx xx',
-  TT: 'xxx-xxxx',
-  TW: 'xxxx-xxxxxx',
-  TZ: 'xxx xxx xxxx',
-  UA: 'xx xxx xx xx',
-  UG: 'xxx xxxxxxx',
-  US: 'xxx-xxx-xxxx',
-  UY: 'xxx-xxxxx',
-  UZ: 'xxx-xxx-xx-xx',
-  VC: 'xxx-xxxx',
-  VE: 'xxxx-xxx-xxxx',
-  VN: 'xxxx-xxxxxxx',
-  YE: 'xxxx-xxxx',
-  ZA: 'xxx-xxx-xxxx',
-  ZM: 'xxx-xxxxxxx',
-  ZW: 'xx xxx xxxx',
-  KW: 'xxx xx xxxx',
-  BH: 'xxxx xxxx',
-  AG: 'xxx-xxxx',
-  AI: 'xxx-xxxx',
-  AS: 'xxx-xxxx',
-  DM: 'xxx-xxxx',
-  GD: 'xxx-xxxx',
-  GU: 'xxx-xxxx',
-  KN: 'xxx-xxxx',
-  LC: 'xxx-xxxx',
-  MP: 'xxx-xxxx',
-  MS: 'xxx-xxxx',
-  SX: 'xxx-xxxx',
-  TC: 'xxx-xxxx',
-  VG: 'xxx-xxxx',
-  VI: 'xxx-xxxx',
-  GR: 'xx xxx xxxx',
-  BE: 'xxx xx xx xx',
-  FR: 'xx xx xx xx xx',
-  ES: 'xxx xxx xxx',
-  IT: 'xxx xxxx xxxx',
-  VA: 'xxx xxxx xxxx',
-  AT: 'xxx xxxxxxxx',
-  GG: 'xxxx xxxx',
-  IM: 'xxxx xxxx',
-  JE: 'xxxx xxxx',
-  SJ: 'xxx xx xxx',
-  DE: 'xxx xxxxxxxx',
-  BR: 'xx xxxx-xxxx',
-  CL: 'x xxxx xxxx',
-  CC: 'xxx xxxx xxxx',
-  CX: 'xxx xxxx xxxx',
-  JP: 'xx xxxx xxxx',
-  KR: 'xx xxxx xxxx',
-  AF: 'xx xxx xxxx',
-  IR: 'xx xxxx xxxx',
-  EH: 'xx xx xx xx',
-  LY: 'xx xxxxx xxxx',
-  MR: 'xx xx xx xx',
-  ML: 'xx xx xx xx',
-  GN: 'xx xx xx xx',
-  CI: 'xx xx xx xx',
-  BF: 'xx xx xx xx',
-  NE: 'xx xx xx xx',
-  BJ: 'xx xx xx xx',
-  TD: 'xx xx xx xx',
-  CF: 'xx xx xx xx',
-  CM: 'xx xx xx xx',
-  CV: 'xxx xxxx',
-  ST: 'xx xxx xx',
-  GQ: 'xx xxx xxxx',
-  GA: 'xx xx xx xx',
-  CG: 'xx xx xx xx',
-  CD: 'xx xxx xxxx',
-  AO: 'xx xxx xxxx',
-  GW: 'xx xx xx xx',
-  IO: 'xxx xxxx',
-  AC: 'xxxx',
-  SD: 'xx xxx xxxx',
-  DJ: 'xx xx xx xx',
-  BI: 'xx xx xx xx',
-  RE: 'xxx xx xx xx',
-  YT: 'xxx xx xx xx',
-  KM: 'xx xx xx xx',
-  SH: 'xxxx',
-  TA: 'xxxx',
-  ER: 'x xxx xxxx',
-  FO: 'xxx xxx',
-  GL: 'xx xx xx',
-  IE: 'xx xxx xxxx',
-  IS: 'xxx xxxx',
-  MT: 'xx xx xx xx',
-  CY: 'xx xxxxxx',
-  FI: 'xx xxx xxxx',
-  AX: 'xxx xxx',
-  AD: 'xxx xxx',
-  MC: 'xx xx xx xx',
-  XK: 'xx xxx xxxx',
-  LI: 'xxx xx xx',
-  PM: 'xxx xxxx',
-  GP: 'xxx xx xx xx',
-  MF: 'xxx xx xx xx',
-  GF: 'xxx xx xx xx',
-  MQ: 'xxx xx xx xx',
-  NF: 'x xxxx',
-  NR: 'xxx xxxx',
-  TO: 'xxxxx',
-  SB: 'xxxxx',
-  VU: 'xx xxxxx',
-  PW: 'xxx xxxx',
-  WF: 'xx xxxx',
-  CK: 'xx xxx',
-  NU: 'xxxx',
-  WS: 'xx xxxxx',
-  KI: 'xx xxx',
-  NC: 'xx xx xx',
-  TV: 'xxxxx',
-  PF: 'xx xx xx xx',
-  TK: 'xxxx',
-  FM: 'xxx xxxx',
-  MH: 'xxx xxxx',
-  KP: 'xx xxx xxxx',
-  LB: 'xx xxx xxx',
-  JO: 'xx xxxx xxxx',
-  SY: 'xx xxxx xxxx',
-  IQ: 'xx xxx xxxx',
-  BT: 'x xxx xxx',
-  TM: 'x xxx xxx',
-  AZ: 'xx xxx xx xx',
-  GE: 'xxx xxx xxx',
-  AQ: 'xx xxxx',
-  BG: 'xx xxx xxxx',
-  EE: 'xxx xxxx',
-  BY: 'xx xxx-xx-xx',
-  BA: 'xx xxx-xxxx',
-  FK: 'xxxxx',
-  BL: 'xxx xx xx xx',
-  EC: 'xx xxx xxxx',
-  CW: 'xxx xxxx',
-  BQ: 'xxx xxxx',
+  IN: 'xxxx xxxxxx', // India, Example: 7394926646 formatted to 7394 926646
```

Comment:
What is the reason for country codes getting reduced in new changes ?

Reply:
CC: Cocos (Keeling) Islands take this for example, this has been removed it's a remote territory of Australia in the Indian Ocean. Similarly there were a bunch of country codes which were either uninhabited or were not recognised or were merged into a single country. So removing those.